modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must specify the projects associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same project as the `ServiceMeshControlPlane` resource and name it `default`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are a part of the `ServiceMeshMemberRoll` resource.
+====
+
+[NOTE]
+====
+The member projects are only updated if the {ProductShortName} control plane installation succeeds.
+====
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll`.
+
+* `ServiceMeshMemberRoll` resources are reconciled in response to the following events:
```

Comment:
Not sure if this section is needed. As a user, I expect the SMMR to be reconciled whenever it needs to be, just like any other resource. I defined the desired state in the SMMR and I expect the operator to _always_ try to bring the actual state to how I want it to be. Thus, going into detail of when the operator reconciles the SMMR seems unnecessary. Also, when introducing the SMCP resource, we never explain when it gets reconciled.

Reply:
Removed.